.github/workflows/test.yaml

```diff
@@ -81,10 +81,11 @@ jobs:
 
       - name: Upload test results
         if: ${{ failure() }}
-        uses: actions/upload-artifact@v3
+        uses: actions/upload-artifact@v2
```

Comment:
What's the problem with `upload-artifact@v3`?

Reply:
Just temporary debugging `CI` tests